sdks/go/pkg/beam/runners/prism/internal/coders.go

```diff
@@ -187,6 +187,30 @@ func reconcileCoders(bundle, base map[string]*pipepb.Coder) {
 	}
 }
 
+func reconcileWindowCoders(bundle, base map[string]*pipepb.Coder, ws map[string]*pipepb.WindowingStrategy) {
```

Comment:
I might open a separate PR for this, but this happens because the coder passed through the instruction is not included when there are multiple windowing strategies.